### PR TITLE
refoctors building definitions for body params

### PR DESCRIPTION
**Pull request by LeFnord**

### CODE REVIEW

File: spec/lib/move_params_spec.rb [hunk 1]
```diff
@@ -7,50 +7,6 @@
...
-      specify { expect { |b| subject.find_post_put(paths, &b) }.to yield_successive_args *expected }
-    end
-  end
-
-  describe 'find_definition_and_params' do
-    specify do
-      subject.instance_variable_set(:@definitions, definitions)
-      subject.find_definition_and_params(found_path[:post], :post)
-      expect(definitions.keys).to include 'InBody'
-      expect(definitions['postRequestInBody'].keys).to_not include :description
```
Comment (texpert) [inline spec/lib/move_params_spec.rb]: Wouldn't this case be lost?

Comment (LeFnord) [inline spec/lib/move_params_spec.rb]: in principal yeap … cause now not the whole set of paths and definitions are the parameters of the method → see here https://github.com/ruby-grape/grape-swagger/pull/467/files#diff-7af39528837333c960830618fb6a52e9R162

maybe I should rename it

Comment (LeFnord) [inline spec/lib/move_params_spec.rb]: sorry … first comment was wrong an dnot related,

no it is always there, see: https://github.com/ruby-grape/grape-swagger/pull/467/files#diff-0a42864b10ca75eb0af2c0fc45139eb0R25

Comment (LeFnord) [inline spec/lib/move_params_spec.rb]: but yeah should adding a spec to proof it, thanks

Comment (texpert) [inline spec/lib/move_params_spec.rb]: Yep, I remember it from here https://github.com/ruby-grape/grape-swagger/pull/450/files :)

Comment (LeFnord) [inline spec/lib/move_params_spec.rb]: yeap … re-added it, thanks 😄

**Pull request merged**